### DISCUSSION TITLE
A way to have both `local` and `federated` in the same feed?

### DISCUSSION THREAD

**Discussion by Geobert** [cheeaun/phanpy#562] [Q&A] [answered]
Is there a way to have the `All` feed we have on Mastodon that merges `Local` and `Others`?

**Comment by cheeaun** [comment, marked as answer]
Hmm you found a bug. 'Federated' should be Local + Remote. Accidentally introduced this bug when implementing some APIs for Pixelfed. Will fix 🙇‍♂️

**Comment by Geobert** [comment]
Do you want an issue to track it or this thread is good enough?

**Comment by Geobert** [comment]
Fixed on dev
